src/main/resources/brapi/calls.json

```diff
@@ -67,6 +66,21 @@
           "1.2"
         ]
       },
+      {
+        "call": "observationunits",
```

Comment:
Why is there a new endpoint added here?

Reply:
/observationunits is the new endpoint for phenotypes-search (GET) in [v1.3.](https://brapi.docs.apiary.io/#reference/phenotypes/phenotypes-search). This should be GET not post. The replacement for phenotypes-search (POST) is /search/observationunits. Added this before to pass succeeding observationunit calls. I will remove this endpoint since it's not yet supported.